emailproxy.py

```diff
@@ -570,6 +570,79 @@ def _save_cache(cache_store_identifier, output_config_parser):
             Log.error('Error saving state to cache store file at', cache_store_identifier, '- is the file writable?')
 
 
+class Cryptographer:
+    ITERATIONS = 870000  # taken from cryptography's suggestion of using Django's defaults
+    LEGACY_ITERATIONS = 100000
```

Comment:
What do you think about keeping a list of previous iteration counts given that the recommendation changes fairly often? (e.g., from 480,000 in December 2022 to 870,000 under a year later)

Reply:
Well this change stores the number of iterations in the config, so right now the options it tries are: the initial default 100_000, if there's a value in the config, and the new default 870_000, going forward if the config is updated it will store the value in the config, so it should really never go past that and the legacy value is just for historical reasons.

Ideally it's just trying [ITERATIONS, configured_iterations] and if the config is up to date that's actually just [ITERATIONS] when uniqued.